examples/orchestrator/README.md

```diff
@@ -0,0 +1,95 @@
+# orchestrator
+
+orchestrator is a highly available, administrative, and visual tool for managing MySQL replication topologies.
```

Comment:
Orchestrator is a MySQL high availability and replication management tool, runs as a service and provides command line access, HTTP API and Web interface.